test/utils/fixtures/init-config/default-config.yaml

```diff
@@ -56,9 +56,11 @@ publisher:
 # The parameters property represents IG.definition.parameter. Rather
 # than a list of code/value pairs (as in the ImplementationGuide
 # resource), the code is the YAML key. If a parameter allows repeating
-# values, the value in the YAML should be a sequence/array. For a
-# partial list of allowed parameters see:
-# https://confluence.hl7.org/display/FHIR/Implementation+Guide+Parameters
+# values, the value in the YAML should be a sequence/array. 
```

Comment:
The test for the default config file is failing because the fixture has an extra space after line 59. Removing the space should fix the test.
```suggestion
# values, the value in the YAML should be a sequence/array.
```

Reply:
Indeed 😳